### ISSUE TITLE
All tests are skipped?

### ISSUE THREAD

**Issue by ToastyMallows** [salcido/discogs-enhancer#130]
In [this commit](https://github.com/salcido/discogs-enhancer/commit/0c7f17c08b4b932abd5d09c60f902f019545cd47), all the tests were changed to add `.skip()`, making all tests "Pending".  Is this as intended?  Is there something wrong with the tests?

I enabled one of them locally and it looks like it is passing.

**Comment by salcido** [comment]
@ToastyMallows Yeah, that's intentional. For some reason I can't get the tests to pass reliably when the builds run with Travis. But they will pass when running locally. I got tired of having to re-run the tests 4 or 5 times to get a successful build so I just disabled them as a work around. 

I spent some time trying to diagnose the issue but never figured it out. I thought it might be due to rate limiting on Discogs so [I slowed down the tests](https://github.com/salcido/discogs-enhancer/blob/master/test/test.js#L10) but that didn't seem to work. I'm kind of out of ideas at this point.
